frontend/packages/kubevirt-plugin/src/components/create-vm/forms/VMHW.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+import { Button, ButtonVariant, Popover, PopoverPosition } from '@patternfly/react-core';
+import { useTranslation } from 'react-i18next';
+
+import './create-vm-form.scss';
+
+type VMIPProps = {
+  data: string[];
+  title?: string;
+};
+
+const VMHW: React.FC<VMIPProps> = ({ data, title }) => {
+  const { t } = useTranslation();
+  if (!data) {
+    return <>{t('kubevirt-plugin~None')}</>;
+  }
+
+  const isMoreThenOneHW = data?.length > 1;
```

Comment:
btw/ it's a nice pattern to have a list with (+ N objects ) button as a common component